src/main/java/me/zeroeightsix/kami/manager/mangers/FriendManager.kt

```diff
@@ -12,7 +12,7 @@ import java.io.*
  * Created by Xiaro on 26/08/20
  */
 object FriendManager {
-    var friendFile = FriendFile(true, ArrayList())
+    var friendFile = FriendFile()
```

Comment:
Why was this change made??

Reply:
They probably rebased master instead merging it

Reply:
Sorry, I dont know the difference lol